src/ReactiveUI.XamForms/ViewModelViewHost.cs

```diff
@@ -70,7 +89,9 @@ public ViewModelViewHost()
                 (vm, contract) => new { ViewModel = vm, Contract = contract, });
 
             var platform = Locator.Current.GetService<IPlatformOperations>();
-            if (platform == null) {
+
+            if (platform == null)
+            {
```

Comment:
Is there an extra newline here? (I think my rebracer at least is pro `{` on the same line)